CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Contributing
+sidebar:
+  order: 8
+---
+
+There are many ways to contribute to the Data Package standard. Here are a few:
+
+- **Use the standard**: The best way to contribute is to use the standard in your projects and provide feedback on what works and what doesn't.
+
+- **Spread the word**: Help us spread the word about the Data Package standard. The more people who know about it, the more people who can contribute.
+
+- **Participate in discussions**: Share your ideas and feedback in the [Data Package Discusions](https://github.com/frictionlessdata/datapackage/discussions). We use a voting system to prioritize issues and feature requests.
+
+- **Report issues**: If you find a bug or have a feature request, please report it in the [GitHub issue tracker](https://github.com/frictionlessdata/datapackage/issues). There a few predefined issue templates to help you get started.
+
+- **Contribute to the repository**: You can contribute to the Data Package standard repository. Please read this guide for more details on the process.
+
+## Research and development
+
+The Data Package standard is developed using a research and development model. This means that the standard is developed in an iterative way, with new features and changes being proposed, discussed, and implemented in a collaborative way.
+
+There are two main feedback trackers for the Data Package standard:
+
+- **[Data Package Discussions](https://github.com/frictionlessdata/datapackage/discussions)**: This is where general discussions and new feature proposals to the Data Package standard take place. You can use this forum to share your ideas, ask questions, and provide feedback. Using discussions and the voting mechanism, the community and the Data Package Working Group can help prioritize issues and feature requests. The discussions is a good place to design and share a feature draft or implementation details. Generally speaking, this is a research phase.
```

Comment:
```suggestion
- **[Data Package Discussions](https://github.com/frictionlessdata/datapackage/discussions)**: This is where general discussions and new feature proposals to the Data Package standard take place. You can use this forum to share your ideas, ask questions, and provide feedback. Using discussions and the voting mechanism, the community and the Data Package Working Group can help prioritize issues and feature requests. The discussions is a good place to design and share a feature draft or implementation details. Generally speaking, this is the research phase.
```

I read this as: of the research and development model, this is the first phase, the research phase.